src/clj/game/cards/resources.clj

```diff
@@ -2619,12 +2619,13 @@
                 :effect (effect (update! (dissoc card :supplier-installed)))}]})
 
    "The Turning Wheel"
-   (let [ttw-ab (fn [m s]
-                  {:label (str "Access an additional card in " m)
-                   :cost [:power 2]
-                   :req (req run)
-                   :msg (msg "access 1 additional card from " m " for the remainder of the run")
-                   :effect (req (access-bonus state side s 1))})]
+   (letfn [(ttw-ab [name server]
+            {:label (str "Access an additional card in " name)
+            :cost [:power 2]
+            :req (req run)
+            :msg (msg "access 1 additional card from " name " for the remainder of the run")
+            :effect (req (access-bonus state side server 1))})
+          ]
```

Comment:
```suggestion
                  (ttw-bounce [name server]
                   :label (str "Bounce " name)
                   :cost [:click 1]
                   :effect (req (add-counter state side card :power 1)
                                (swap! state update-in [:runner :register :unsuccessful-run] #(conj % server))
                                (swap! state assoc-in [:run :unsuccessful] true)
                                (system-msg state :runner (str "places a power counter on " (:title card))))}]
```

You can define the entire thing as a function to avoid writing the same code twice below.